### DISCUSSION TITLE
Issue with Article Publish Time Metadata in Obsidian Web

### DISCUSSION THREAD

**Discussion by LiangSam** [coddingtonbear/obsidian-web#159] [Bug Report]
### Describe the bug

Hello Obsidian Web Clipper Developers,

I hope this message finds you well.

I am writing to report an issue I am encountering with the Web Clipper plugin when saving articles to my Obsidian vault. The article's publish time metadata, which is present on the webpage, does not appear in the exported notes.

The webpage with the metadata is: [Article Link](https://fortelabs.com/blog/para/)

Here is the relevant HTML meta tag from the webpage source:

```html
<meta property="article:published_time" content="2023-02-24T16:29:17+00:00">
```

I am using the following template for my exported notes:

```markdown
---
page-title: {{json page.title}}
url: {{page.url}}
date: "{{date}}"
publish-time: {{article.published_time}}
Author: {{article.byline}}
---
```

Despite checking the plugin settings and template configuration, the "publish-time" field remains empty in the exported notes. I am unsure if this is a known issue or if there are specific configurations that need to be adjusted to correctly capture the publish time.

I would appreciate any guidance or assistance you could provide to resolve this issue. 

Thank you in advance for your time and attention.

### To Reproduce

2. Click on '...'
3. Scroll down to '...'

### Operating System

windows

### Browser

edge

### Browser Version

122.0.2348.0 dev

**Comment by coddingtonbear** [comment]
I'm afraid that's not supported!  I'm sort of curious why you might have came to the conclusion that it was, though?

**Comment by trailsnorth** [comment]
I understand this is not currently supported, however I am curious if published date/time is on your roadmap?
